text/0803-type-ascription.md

```diff
@@ -161,7 +161,7 @@ as an lvalue (i.e., no new temporary), then there is potential for unsoundness:
 ```
 let mut foo: S = ...;
 {
-    let bar = &mut (foo: T);  // S <: T, no coercion required
+    let bar = &mut (foo: T);  // S <: T, coercion from S to T
```

Comment:
In case you wonder: "no coercion required" was wrong because [this is one type of coercion](https://doc.rust-lang.org/reference/type-coercions.html#coercion-types).